### PR TITLE
y not?

### PR DESCRIPTION
**Pull request by ladyada**
v qt!

### CODE REVIEW

File: ports/espressif/boards/adafruit_qtpy_esp32s2/mpconfigboard.h [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#define MICROPY_HW_NEOPIXEL (&pin_GPIO39)
+#define CIRCUITPY_STATUS_LED_POWER (&pin_GPIO38)
+
+#define CIRCUITPY_BOOT_BUTTON (&pin_GPIO0)
+
+#define BOARD_USER_SAFE_MODE_ACTION translate("pressing boot button at start up.\n")
+
+#define AUTORESET_DELAY_MS 500
+
+#define DEFAULT_I2C_BUS_SCL (&pin_GPIO7)
```
Comment (gamblor21) [inline ports/espressif/boards/adafruit_qtpy_esp32s2/mpconfigboard.h:41]: SCL/SDA are reversed from pins.c

Comment (ladyada) [reply ports/espressif/boards/adafruit_qtpy_esp32s2/mpconfigboard.h]: !!! fixed

**Review by gamblor21 — DISMISSED** [review]
LGTM!

**Review by jepler — APPROVED** [review]
If it builds, ship it!

**Pull request merged**